Cargo.toml

```diff
@@ -113,50 +113,73 @@ rustdoc-args = ["--cfg", "docsrs", "--generate-link-to-definition"]
 
 [package.metadata.release]
 pre-release-replacements = [
-  {file="CHANGELOG.md", search="Unreleased", replace="{{version}}", min=1},
-  {file="CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}", exactly=1},
-  {file="CHANGELOG.md", search="ReleaseDate", replace="{{date}}", min=1},
-  {file="CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n## [Unreleased] - ReleaseDate\n", exactly=1},
-  {file="CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[Unreleased]: https://github.com/rust-cli/config-rs/compare/{{tag_name}}...HEAD", exactly=1},
+  { file = "CHANGELOG.md", search = "Unreleased", replace = "{{version}}", min = 1 },
+  { file = "CHANGELOG.md", search = "\\.\\.\\.HEAD", replace = "...{{tag_name}}", exactly = 1 },
+  { file = "CHANGELOG.md", search = "ReleaseDate", replace = "{{date}}", min = 1 },
+  { file = "CHANGELOG.md", search = "<!-- next-header -->", replace = "<!-- next-header -->\n## [Unreleased] - ReleaseDate\n", exactly = 1 },
+  { file = "CHANGELOG.md", search = "<!-- next-url -->", replace = "<!-- next-url -->\n[Unreleased]: https://github.com/rust-cli/config-rs/compare/{{tag_name}}...HEAD", exactly = 1 },
 ]
 
 [features]
-default = ["toml", "json", "yaml", "ini", "ron", "json5", "convert-case", "async"]
+default = [
+  "toml",
+  "json",
+  "yaml",
+  "ini",
+  "ron",
+  "json5",
+  "convert-case",
+  "async",
+]
 json = ["serde_json"]
 yaml = ["yaml-rust2"]
 ini = ["rust-ini"]
 json5 = ["json5_rs", "serde/derive"]
 convert-case = ["convert_case"]
-preserve_order = ["indexmap", "toml?/preserve_order", "serde_json?/preserve_order", "ron?/indexmap"]
+preserve_order = [
+  "indexmap",
+  "toml?/preserve_order",
+  "serde_json?/preserve_order",
+  "ron?/indexmap",
+]
 async = ["async-trait"]
 toml = ["dep:toml"]
 
 [dependencies]
 serde = "1.0"
 
 async-trait = { version = "0.1", optional = true }
-toml = { version = "0.9", optional = true, default-features = false, features = ["parse", "serde"] }
+toml = { version = "0.9", optional = true, default-features = false, features = [
+  "parse",
+  "serde",
+] }
 serde_json = { version = "1.0", optional = true }
 yaml-rust2 = { version = "0.10", optional = true }
 rust-ini = { version = "0.21", optional = true }
-ron = { version = "0.8", optional = true }
+ron = { version = "0.10", optional = true }
```

Comment:
We are not ready to upgrade ron, see https://github.com/rust-cli/config-rs/pull/476#pullrequestreview-2698875901